packages/portal/backend/src/Factory.ts

```diff
@@ -25,21 +27,24 @@ export class Factory {
      * @returns {IREST}
      */
     public static async getCustomRouteHandler(name?: string): Promise<IREST> {
-        if (typeof name === 'undefined') {
+        if (typeof name === "undefined") {
             name = Factory.getName();
         }
         try {
             // NOTE: using require instead of import because file might not be present in forks
             // import complains about this, but require does not.
             let plug: any;
-            if (name === 'classytest') {
+            const pluginPath = Config.getInstance().getProp(ConfigKey.plugin_fullpath);
+            Log.info("Factory::getCustomRouteHandler() - plugin path: " + pluginPath);
+            if (name === Factory.TESTNAME || typeof pluginPath === "undefined") {
                 Log.info("Factory::getCustomRouteHandler() - instantiating DefaultCourseRoutes for: " + name);
-                plug = await require('./custom/DefaultCourseRoutes'); // default for testing
+                plug = await require("./custom/DefaultCourseRoutes"); // default for testing
```

Comment:
Is this file a TestingCourseRoutes file or a DefaultCourseRoutes file? Or Both? Also, does it belong in the custom folder?

Reply:
Perhaps the default or testing files should be namespaced apart from their custom counterparts.